go/test/endtoend/transaction/twopc/schema.sql

```diff
@@ -18,4 +18,27 @@ create table twopc_t1
     id  bigint,
     col bigint,
     primary key (id)
-) Engine=InnoDB;
\ No newline at end of file
+) Engine=InnoDB;
+
+create table twopc_lookup
+(
+    id  bigint,
+    col bigint,
+    col_unique bigint,
+    primary key (id)
+) Engine=InnoDB;
+
+create table lookup
+(
+    col         varchar(128),
+    id          bigint,
+    keyspace_id varbinary(100),
+    primary key (id)
+) Engine = InnoDB;
```

Comment:
if this is non unique vindex, ideally the primary key should be `(col, id)` otherwise this becomes a unique vindex even though VTGates will treat it as non-unique.

Reply:
Why? id is always going to guarantee uniqueness. Isn't that the only reason why we add it to the table?